### PR TITLE
[ZH] Add `Game mode/osu!catch`

### PR DESCRIPTION
**Pull request by FlyingFish23**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [ ] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

才刚玩 osu!catch 不久，有些专有名词参考了 osu!catch 的 RC，如果老玩家有更常见的译名，欢迎告诉我！

### CODE REVIEW

File: wiki/Game_mode/osu!catch/zh.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+- 休息时间的 Catcher 仍然能左右移动。
+- Miss 后 Catcher 会有一个小动画，类似于 osu!taiko pippidon 的 Miss 动画。
+- 第一个默认 Catcher 形象是修改过的，[应援团系列](https://en.wikipedia.org/wiki/Osu!_Tatakae!_Ouendan)中的 Ryuuta Ippongi。直到 2014 年才被更改成现在的 Catcher 形象和模式吉祥物，[Yuzu](/wiki/Mascots#yuzu)。
+- 当 [Auto](/wiki/Game_modifier/Auto) 游玩时，玩家名称将会是 *salad!*。
+
+### History
+
+![特殊模式中的 osu!catch 图标](/wiki/shared/Ctb_logo.jpg "Special Mode 中的 Catch the Beat 图标")
+
+- 在 2012 年 4 月 10 号前，osu!catch 模式的谱面都不能被 Rank 或包含在 Ranked 谱面当中。但现在有了变化，谱师可以制作 osu!catch 模式的谱面，跟随相关规则即有机会 Rank。目前规则的制定还在起步阶段，但是随着 osu!catch 玩家的反馈和 Ranked 谱面越来越多，规则也将会慢慢成长，最终和 osu! 模式的规则一样充分具体。
```
Comment (YumeMuzi) [inline wiki/Game_mode/osu!catch/zh.md:115]: 目前我暂且把 Ranked 往 上架 翻译，可以不用留类似的英文原词

File: wiki/Game_mode/osu!catch/zh.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+- 第一张包含至少一个 osu!catch 谱面的 Ranked 谱面为 [Yousei Teikoku - Dare so Ka no Gekka (TV Size) (NatsumeRin)，osu!catch 难度由 Uran 客串制作](https://osu.ppy.sh/beatmapsets/13676)。
+- 第一张 Rank 的仅 osu!catch 模式谱面为 [Rita - Hajimari no Toki (Deif)](https://osu.ppy.sh/beatmapsets/91485)。
```
Comment (YumeMuzi) [inline wiki/Game_mode/osu!catch/zh.md:117]:
```suggestion
- 第一张上架的 osu!catch 专谱为 [Rita - Hajimari no Toki (Deif)](https://osu.ppy.sh/beatmapsets/91485)。
```

File: wiki/Game_mode/osu!catch/zh.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
+<!-- TODO: needs a grammar/style check -->
+
+# osu!catch
+
+![osu!catch 游戏界面](/wiki/shared/catch-gameplay.jpg "osu!catch 界面")
+
+osu!catch，曾经也被称为 *Catch the Beat* 或 *CtB*。在此模式下玩家需要控制一个小人（Catcher）随着节奏接取从天而降的水果。
```
[the remark this answers is not in the document]
Comment (FlyingFish23) [reply wiki/Game_mode/osu!catch/zh.md:7]: 目前这里还是保留原文吧，感觉这两种翻译都不太能够完美的融入语言当中，个人认为如果要对这个词翻译，还是参考ctb玩家的意见，不过谢谢你的建议！

File: wiki/Game_mode/osu!catch/zh.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+
+## 小知识
+
+### 游玩
+
+- [Auto](/wiki/Game_modifier/Auto) *永远*会接住全部的香蕉。
+- 如果启用了`打击偏差`，则在 osu!catch 中使用模式一直是`色块`。
+- 休息时间的 Catcher 仍然能左右移动。
+- Miss 后 Catcher 会有一个小动画，类似于 osu!taiko pippidon 的 Miss 动画。
+- 第一个默认 Catcher 形象是修改过的，[应援团系列](https://en.wikipedia.org/wiki/Osu!_Tatakae!_Ouendan)中的 Ryuuta Ippongi。直到 2014 年才被更改成现在的 Catcher 形象和模式吉祥物，[Yuzu](/wiki/Mascots#yuzu)。
```
[the remark this answers is not in the document]
Comment (FlyingFish23) [reply wiki/Game_mode/osu!catch/zh.md:108]: 这里的英文原文为chibified，不是替换的意思，我也不太清楚这个词的意思，所以暂且用了"修改"这个词
谷歌了一下说这个词的意思是这样的：
截屏2022-12-22 20 10 11

**Comment by TicClick** [comment]
@FlyingFish23 If everything is applied, please resolve YumeMuzi's comments, if not, please post objections and work it out further with them

**Comment by FlyingFish23** [comment]
> @FlyingFish23 If everything is applied, please resolve YumeMuzi's comments, if not, please post objections and work it out further with them

have replied and waiting for his response on some details that needs to be discussed, and just resolved conversations that were addressed

**Comment by TTsdzb** [comment]
chibi 在画风上有 Q 版的意思，chibified 在这里感觉像是 “以 Q 版画风画的” 的意思

**Comment by TicClick** [comment]
so, where are we? does this need any further updates?

**Comment by YumeMuzi** [comment]
> so, where are we? does this need any further updates?

it's need to make a deeper review. i'll finish it tmr

File: wiki/Game_mode/osu!catch/zh.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (YumeMuzi) [reply wiki/Game_mode/osu!catch/zh.md:7]: 我就是 ctb 玩家呀...也在 cn ctb community 里

File: wiki/Game_mode/osu!catch/zh.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (YumeMuzi) [reply wiki/Game_mode/osu!catch/zh.md:108]: 我没看原文。原来你把chibi翻译成了`修改过`。这其实是不对的。
chibi (ちび)，有小孩，小家伙的意思，对应到国内应该是`萌化小人`形象。
我把 catcher 翻译成 `接水果小人`，正好有`小`一义。
所以请修改。

I didn't review the original article. you translated `chibi` to `modified`, that's not correct.
chibi, meaning `child`, `kiddy`, or `chico`.
imo my translation `a kiddy who is catching the fruit`, is better than yours.
plz fix.

File: wiki/Game_mode/osu!catch/zh.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (YumeMuzi) [reply wiki/Game_mode/osu!catch/zh.md:7]: 大多数国内玩家还是习惯于叫它`盘子` (plate)，我认为改成盘子可能会丢失 catcher 的原意，所以我建议翻译成接水果小人。
plz fix.

reference: https://yumemuzi.s-ul.eu/emotions/g0eTEJ7K

**Review by YumeMuzi — CHANGES_REQUESTED** [review]
catcher -> 接水果小人
chibi -> 小人形象

**Comment by FlyingFish23** [comment]
Sorry I don't have time to work on this anymore at this time, you guys can grab the source text and start a separate PR (please mention my name) or directly make changes here. Thanks for all the reviews, wish you guys a happy new year!

**Comment by FlyingFish23** [comment]
> for convenience, maybe it's better to close this PR and reopen a new PR?

Go ahead, thanks for all your reviews! They're very detailed and very valuable to me!

**Pull request closed without merging**